### PR TITLE
EIP-1559 support

### LINKED ISSUE #39: EIP-1559 transaction

### PR DESCRIPTION
**Pull request by 34x4p08**
Closes #39

### CODE REVIEW

**Comment by mate-from-mattr** [comment]
Thank you @34x4p08 I will review this tonight

File: src/lib.rs [hunk 1]
```diff
@@ -700,7 +828,7 @@ mod test {
 
     #[test]
     fn test_zero_legacy_002_hash() {
-        run_hash_test::<AccessListTransaction>("./test/zero_eip_2718_002.json");
```
Comment (synlestidae) [inline src/lib.rs:703]: Whoops!

Comment (34x4p08) [reply src/lib.rs:703]: I fixed this and other similar misconfigs in tests

**Comment by 34x4p08** [comment]
Hello @synlestidae!
Do you have any obstacles that prevents you of merging this PR?

**Comment by synlestidae** [comment]
Hey, I’m sorry, the only obstacle is time. I just need to audit the PR
carefully, merge, and deploy. Apologies for the delay.

**Review by synlestidae — APPROVED** [review]

**Comment by synlestidae** [comment]
All done now @34x4p08 ! Your GitHub is now on the README. Thank you so much for contributing.

**Comment by 34x4p08** [comment]
> All done now @34x4p08 ! Your GitHub handle is now on the README. Thank you so much for contributing.

Cool, thanks 👍

**Pull request merged**